frontend/src/metabase/common/components/DatePicker/SpecificDatePicker/SpecificDatePicker.unit.spec.tsx

```diff
@@ -38,11 +38,11 @@ describe("SpecificDatePicker", () => {
   });
 
   it('should be able to set "on" filter', () => {
-    const { onChange } = setup();
+    const { onChange } = setup({ isNew: true });
```

Comment:
`setup` is always used with `isNew: true` - we could just change the default value, or remove the parameter